### DISCUSSION TITLE
卡片复习时显示音标/阅读模式下的段落问题

### DISCUSSION THREAD

**Discussion by ninoson94** [guopenghui/obsidian-language-learner#108] [Ideas]
大佬您好，之前我只能通过划词助手插件搭配anki复习来实现背单词，最近了解到您开发的插件，终于能实现在obsidian中有原句划词的背单词功能了，实在感谢。使用过程中遇到的问题：

1. 能在复习卡片中把音标也显示上去吗？
2.目前在阅读模式下只能显示每一个段落，有时候不小心点了完成阅读，直接跳到新段落，没办法返回去。

**Comment by guopenghui** [comment]
复习卡片现在比较简单，受限于联动的插件，不好实现很丰富的功能，后面可能会自己写一套复习系统

是在电脑上还是手机？ 阅读模式下面有分页栏，可以前进后退进行换页

**Reply by ninoson94** [reply]
底下看到可以换页了，谢谢大佬！ 期待以后的功能～～
